bucket/direnv.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://direnv.net",
+    "checkver": {
+        "github": "https://github.com/direnv/direnv"
+    },
+    "license": "MIT",
+    "version": "2.19.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/direnv/direnv/releases/download/v2.19.0/direnv.windows-amd64.exe",
+            "hash": "14654cb6e4f7bc2e5fa6f5ba0e2568956846ec0ef94c52c6fee09f245f8007cf",
+            "bin": [ [ "direnv.windows-amd64.exe", "direnv" ] ]
+        },
+        "32bit": {
+            "url": "https://github.com/direnv/direnv/releases/download/v2.19.0/direnv.windows-386.exe",
+            "hash": "085b51d85cdec3a98e40a10527a0fb8632d4282435fad7839a659b7d1b1bbf10",
+            "bin": [ [ "direnv.windows-386.exe", "direnv" ] ]
+        }
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-amd64.exe"
```

Comment:
```suggestion
                "url": "https://github.com/direnv/direnv/releases/download/v$version/direnv.windows-amd64.exe#/direnv.exe"
```

Reply:
1. Change these autoupdate properties and run checkver with force parameter.
1. Delete architectures bin and create generic one